src/server/unit_sao.cpp

```diff
@@ -124,6 +124,16 @@ void UnitSAO::sendOutdatedData()
 void UnitSAO::setAttachment(int parent_id, const std::string &bone, v3f position,
 		v3f rotation, bool force_visible)
 {
+	auto *obj = parent_id ? m_env->getActiveObject(parent_id) : nullptr;
```

Comment:
if (auto *obj = (parent_id ? m_env->getActiveObject(parent_id) : nullptr)) can be better :)

Reply:
Same assembly code, same speed, but less readable IMO. Thanks for the suggestion, but I'd prefer to stick to this style.